specifyweb/frontend/js_src/lib/components/Preferences/collectionPreferences.ts

```diff
@@ -30,8 +33,13 @@ collectionPreferences.events.on(
     throttleRate
   )
 );
-collectionPreferences.fetch().catch(softFail);
 
+// Fetch up to date preferences from the back-end
+collectionPreferences
+  .fetch()
+  .catch(() => console.log('Error fetching collection preferences'));
```

Comment:
This is wrong for two reasons:
- It uses `console.log` instead of `console.error`
- It throws away the actual error message, thus it's not helpful

A better would be to have `.catch(error=>console.error(error)`

or even better, make it shorter: `.catch(console.error)`